Bring over these useful docs (#1658)
grafana/k8s-monitoring-helm: charts/k8s-monitoring/docs/ScrapeApplicationMetrics.md

The chart itself provides additional options:

-   `annotationAutodiscovery.extraDiscoveryRules` - Use relabeling rules to filter the Pods or services to scrape.
-   `annotationAutodiscovery.metricsTuning` - Specify which metrics to keep or drop.
-   `annotationAutodiscovery.extraMetricProcessingRules` - Use relabeling rules to process the metrics after scraping
  them.

These values apply to all discovered Pods and services. Refer to the
[feature documentation](../charts/feature-annotation-autodiscovery) to learn about all the possible options.

## Prometheus Operator CRDs

You can use the [Prometheus Operator Objects](../charts/feature-prometheus-operator-objects) feature to detect and
utilize ServiceMonitor, PodMonitor, and Probe objects on the Kubernetes cluster. If any of those objects are detected,
Alloy will utilize them to extend its configuration.

For more information about creating and configuring these options, refer to
the [Prometheus Operator Documentation](https://github.com/prometheus-operator/prometheus-operator).

This chart provides ways to customize how Alloy handles these objects.

### Controlling discovery

> These options in the Helm chart allow for changing how Prometheus Operator objects are discovered:

Reviewer: ```suggestion
To change how Prometheus Operator objects are discovered, use these options in the Helm chart:
```
